test/blackbox-tests/test-cases/reason-expect.t/refmt/refmt.ml

```diff
@@ -0,0 +1,52 @@
+let sprintf = Printf.sprintf
```

Comment:
Instead of copy pasting, what about moving this to `test-cases/utils` and making it available to both tests?

Reply:
Its not an exact copy and paste which makes it awkward. One outputs to a file and the other outputs to stdout.

Reply:
Shouldn't it just copy how the original refmt binary works? The fact that it behaves differently in different tests is wrong